executor/plan_replayer.go

```diff
@@ -180,11 +178,16 @@ func (e *PlanReplayerSingleExec) Next(ctx context.Context, req *chunk.Chunk) err
  |-table_tiflash_replica.txt
  |-variables.toml
  |-bindings.sql
- |-sqls.sql
+ |-sql
+ |   |-sql1.sql
+ |   |-sql2.sql
+ |	 |-....
  |_explain
-     |-explain.txt
+     |-explain1.txt
+     |-explain2.txt
+     |-....
 */
-func (e *PlanReplayerSingleExec) dumpSingle(ctx context.Context, path string) (fileName string, err error) {
+func (e *PlanReplayerExec) dumpSingle(ctx context.Context, path string) (fileName string, err error) {
```

Comment:
Maybe we should change the function name since we dump more than single query now.

Reply:
`fileName = fmt.Sprintf("replayer_single_%v_%v.zip", key, time)` may also be changed.